crates/bdk/tests/wallet.rs

```diff
@@ -445,6 +461,16 @@ macro_rules! check_fee {
     }};
 }
 
+/// A floating point assert! that taker an additional third argument `delta`
+/// as a tolerance value when test for equality the first and second argument.
+macro_rules! assert_delta {
+    ($x:expr, $y:expr, $d:expr) => {
+        if !($x - $y < $d || $y - $x < $d) {
+            panic!();
+        }
+    };
+}
+
```

Comment:
This the way I've found to ignore rounding errors when comparing `FeeRate::from_sat_per_vb` after the wallet signed with PKH descriptor.